spec/models/post_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  user = User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+  subject { Post.new(author: user, title: 'Title', text: 'This is a text') }
+  # it { should belong_to(:author).class_name('User') }
+  # it { should have_many(:comments).foreign_key(:post_id) }
+  # it { should have_many(:likes).foreign_key(:post_id) }
+  # it { should validate_presence_of(:title) }
+  # it { should validate_length_of(:title).is_at_most(250) }
+  # it { should validate_numericality_of(:comments_counter).only_integer.is_greater_than_or_equal_to(0) }
+  # it { should validate_numericality_of(:likes_counter).only_integer.is_greater_than_or_equal_to(0) }
```

Comment:
- [ ] Please, do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.